validate/setup-guides/aws/sg.md

```diff
@@ -123,11 +145,16 @@ Our **inbound rules** will look something like this 👇
 
 Click on **`Save rules`**.
 
+**[REPEAT](#constellation-sg-rules)** This process for ports `9010-9011`.
+
 :::info
-Ports 9000 and 9001 are used for a Public (9000) and Peer-to-Peer (9001) **API** access.   Ports 9000 and 9001 are configurable and can be independent (to your needs) on the Hypergraph Global Layer0 or State Channel network you connect to.  Node Operators will need to learn what ports are opened for access to the State Channels, and update their firewall (change) accordingly
+Ports 9000 and 9001 are used for a Public (9000) and Peer-to-Peer (9001) **API** access.   Ports 9000 and 9001 are configurable and can be independent (to your needs) on the Hypergraph Global Layer0 or State Channel network you connect to.  Node Operators will need to learn what ports are opened for access to the State Channels, and update their firewall (change) accordingly.
+
+The port numbers selected should not matter; however, for non-advanced users, using the ports mentioned in the above section is recommended.
 :::
 
-### MAINNET 2.0 LAUNCH REQUIREMENT
+## MainNet 2.0 Launch Requirements
+
```

Comment:
Can we make "nodectl" below a link to that section? That would help users navigate.